schemas/docfx.json

```diff
@@ -270,6 +270,12 @@
     },
     "warningsAsErrors": {
       "type": "boolean"
+    },
+    "xref": {
```

Comment:
>xref [](start = 5, length = 4)

would `xrefMap` be a better name?

Reply:
use xref as V2

---
In reply to: [218287718](https://github.com/dotnet/docfx/pull/3428#discussion_r218287718) [](ancestors = 218287718)